src/Lecture1.hs

```diff
@@ -81,7 +86,9 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+
+minmax :: Int -> Int -> Int -> Int
+minmax x y z = (max (max x y) z) - (min (min x y) z)
```

Comment:
That's a neat one-liner! 👏🏻 

I suggested using local variables in this function for the sake of practice. But you wrote a very short implementation and I see that you used local variables elsewhere so that's fine 👍🏻 

You can remove some redundant `()` here additionally:

```suggestion
minmax x y z = max (max x y) z - min (min x y) z
```